tests/SortableTest.php

```diff
@@ -371,4 +374,30 @@ public function it_can_tell_if_element_is_last_in_order()
         $this->assertTrue($model[$model->count() - 1]->isLastInOrder());
         $this->assertFalse($model[$model->count() - 2]->isLastInOrder());
     }
+
+    /** @test */
+    public function it_dispatches_event_before_sort()
+    {
+        Event::fake();
+
+        $newOrder = Collection::make(Dummy::all()->pluck('id'))->shuffle()->toArray();
+
+        Dummy::setNewOrder($newOrder);
+
+        // Assert an event was dispatched twice...
+        Event::assertDispatched(Sorting::class);
+    }
+
+    /** @test */
+    public function it_dispatches_event_after_sort()
+    {
+        Event::fake();
+
+        $newOrder = Collection::make(Dummy::all()->pluck('id'))->shuffle()->toArray();
+
+        Dummy::setNewOrder($newOrder);
+
+        // Assert an event was dispatched twice...
```

Comment:
This comment does not seem to be correct

Reply:
✅